libs/components/ag-grid/src/lib/styles/_modern.scss

```diff
@@ -56,8 +56,20 @@
       .ag-header-cell-resize::after {
         mask-size: 1px 2px;
         -webkit-mask-size: 1px 2px;
-        mask-image: url('data:image/gif;base64,R0lGODlhAQACAPAAAAAAAAAAACH5BAEAAAAALAAAAAABAAIAAAICDAoAOw==');
-        -webkit-mask-image: url('data:image/gif;base64,R0lGODlhAQACAPAAAAAAAAAAACH5BAEAAAAALAAAAAABAAIAAAICDAoAOw==');
```

Comment:
Previously this was a 2 pixel gif, one black pixel and one transparent. Recreating the same effect with a linear gradient avoids the `data:` image.